README.md

```
@@ -3,28 +3,102 @@
+# ... 기타 환경변수 (AWS, Firebase, KG이니시스, PortOne 등)
+```
+
+#### 3. IDE에서 실행
+
+- **IntelliJ IDEA**: `BackendApplication` 클래스에서 실행
+- **VS Code**: Spring Boot Dashboard 사용
+- **터미널**: `./gradlew bootRun`
+
+#### 4. 접속
+
+- **API**: http://localhost:8080
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)